Fix android tabbar crashing

drewvolz opened the pull request:
Fixes a crash on all Android tabbed views. They were expecting to receive a hex color string. This uses first readable color (string) instead of that returned instance.

drewvolz committed:
use first readable color instead of object

drewvolz reviewed (commented):
⚠️ This isn't correct yet, working on it...

drewvolz committed:
use strings when tinycolor value is used directly

drewvolz committed:
Update CHANGELOG.md

rye reviewed (approved):
Sounds good to me!

hawkrives reviewed (changes requested):
`toString()` is documented to return differently formatted values depending on the input – can we switch to `toHexString()` (or another format) so we can be sure what we're getting out?

I think RN can handle any of the string formats, but I'd like consistency :-)

drewvolz committed:
opt to use toRgbString everywhere

we have some semi-transparent colors which can be handled by toRgbString
but not toHexString or toString

drewvolz committed:
making flow happy with the type of possibilities
